docs/website/content/docs/v0.1/Configuration/metadata.md

```diff
@@ -4,3 +4,29 @@ weight: 4
 ---
 
 # Metadata
+
+The Metadata server manages the Machine metadata.  Critically, this includes the
+production of the configuration of Talos to be used during the automated installation.
+
+## Talos Machine Configuration
+
+The configuration of each machine is constructed from a number of sources:
+
+  - The `Cluster` of which the `Machine` is a member.
+  - The `ServerClass` which was used to select the `Server` into the `Cluster`.
+  - Any `Server`-specific patches.
+
+The base template is constructed from the Cluster.  Then, any configuration
```

Comment:
Not _quite_ sure how we want to express this, but the base machine config is constructed by the talos bootstrap provider, based on the information present in the cluster manifests that were applied. From there, the patches bit is correct.

Reply:
I was vague there precisely because I didn't know.  ;)

I'll tune it up a bit.